server/status/feed.go

```diff
@@ -68,16 +68,20 @@ func (nef NodeEventFeed) StartNode(desc proto.NodeDescriptor, startedAt int64) {
 // CallComplete is called by a node whenever it completes a request. This will
 // publish an appropriate event to the feed based on the results of the call.
 func (nef NodeEventFeed) CallComplete(args proto.Request, reply proto.Response) {
+	method := args.Method()
```

Comment:
This is going to be misleading. I think we should probably publish a call event for every method in the batch, not just the first method in the batch.